js/package.json

```diff
@@ -97,5 +85,28 @@
     "ts-jest": "^26.5.4",
     "typescript": "^4.2.4",
     "yarn-deduplicate": "^3.1.0"
+  },
+  "resolutions": {
+		"**/@types/react": "^17.0.0"
+  },
+  "sideEffects": [
+    "style/**/*.css",
+    "style/index.js"
+  ],
+  "styleModule": "style/index.js",
+  "jupyterlab": {
+    "discovery": {
+        "server": {
+          "managers": [
+            "pip"
+          ],
+          "base": {
+            "name": "jupyter-fs"
+          }
+        }
+    },
+    "extension": true,
```

Comment:
I assume it looks for `lib/extension.js` by default, but we put the path in other examples e.g. [here](https://github.com/jpmorganchase/jupyterlab_templates/blob/main/js/package.json#L32)

Reply:
Okay, I had to double check this one since I've *never* seen anyone assign anything other than a boolean to `"extension"` here. The code that implements this is [here](https://github.com/jupyterlab/jupyterlab/blob/9b5a6fd63df142424ec785e07993c8586cd7107c/builder/src/extensionConfig.ts#L47-L58):

```typescript
  // Handle the extension entry point and the lib entry point, if different
  const index = require.resolve(packagePath);
  const exposes: { [id: string]: string } = {
    './index': index
  };


  const extension = data.jupyterlab.extension;
  if (extension === true) {
    exposes['./extension'] = index;
  } else if (typeof extension === 'string') {
    exposes['./extension'] = path.join(packagePath, extension);
  }
```

If `"extension"` is just true, it will default to whatever `require.resolve(packagePath)` returns. Since all of the jupyterlab typescript config sets `"moduleResolution": "node"` this will literally give back the path of the entry point that `import "jupyter-fs"` resolves to. In our case this will always be `lib/index.js`.

So I don't think we gain anything here by explicitly setting the extension entry point here.